.github/workflows/build.yaml

```diff
@@ -284,21 +304,4 @@ jobs:
       working-directory: ${{ github.workspace }}/openmw-deps
       run: |
         git remote set-url --push origin "${{ vars.PUSH_URL }}"
-        git push origin vcpkg-x64-${{ matrix.image }}-${{ github.sha }}
-
-    - name: Repackage binaries for symbol server
-      env:
-        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      if: ${{ env.AWS_ACCESS_KEY_ID != '' && env.AWS_SECRET_ACCESS_KEY != '' }}
-      working-directory: ${{ github.workspace }}/openmw-deps/windows
-      run: .\Store-Symbols.ps1 ${{ github.workspace }}\intermediate
-
-    - name: Upload symbols to symbol server
-      env:
-        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        AWS_DEFAULT_REGION: eu-west-3
-      if: ${{ env.AWS_ACCESS_KEY_ID != '' && env.AWS_SECRET_ACCESS_KEY != '' }}
-      working-directory: ${{ github.workspace }}/openmw-deps/windows/SymStore
-      run:  aws --endpoint-url https://rgw.ctrl-c.liu.se s3 sync --size-only --exclude * --include *.dl_ --include *.pd_ . s3://openmw-sym
+        git push -o merge_request.create origin vcpkg-x64-${{ github.sha }}
```

Comment:
Maybe `--push-option` so it's a little clearer which feature we're using as it's not a super common flag people will already know, and googling single-letter flags is generally ineffective.

Reply:
Also, who does the MR say it's owned by? I suspect it's going to say it was me.

Reply:
Changed to `--push-option`. I don't know who will be the owner.

Reply:
Also I've tried to make push work in my repo and gitlab automatically creates MR to upstream. This is not what we need. So I've removed this.